pkg/codegen/docs/gen.go

```diff
@@ -1514,6 +1545,26 @@ func (mod *modContext) genIndex() indexData {
 	return data
 }
 
+<<<<<<< HEAD
```

Comment:
Looks like you have some merge conflict. Pat just merged some changes related to decoding the language info from the schema.